pulp_2to3_migration/app/plugin/rpm/migrator.py

```diff
@@ -320,3 +336,32 @@ def relate_packages_to_module(self, module_dc):
                 thru.append(ModulemdPackages(package_id=pkg.pk, modulemd_id=module_dc.content.pk))
                 already_related.append(pkg.nevra)
         return thru
+
+    def relate_packages_to_group(self, group_dc):
+        """
+        Relate Packages to a Group.
+
+        Args:
+            module_dc (pulpcore.plugin.stages.DeclarativeContent): dc for a PackageGroup
+        """
+        PackageGroupPackages = pulp3_models.PackageGroup.related_packages.through
+        packages = group_dc.content.packages
+        package_list = []
+        for pkg in packages:
+            package_list.append(pkg['name'])
+        pulp2_repo_id = group_dc.extra_data.get('pulp2_repo_id')
+        pulp2_repo = Pulp2Repository.objects.get(pulp2_repo_id=pulp2_repo_id)
+        # all pulp2 unit_ids for rpm within the pulp2repo
+        unit_ids = Pulp2RepoContent.objects.filter(
+            pulp2_repository=pulp2_repo,
+            pulp2_content_type_id='rpm').values_list('pulp2_unit_id', flat=True)
+        # all pulp3 rpm pks within the pulp2repo
+        pulp3_content = set(Pulp2Content.objects.filter(pulp2_id__in=unit_ids).only(
```

Comment:
Is there a real need for ```pulp3_content``` to be a set?  It's best to keep it a queryset in most cases, because sometimes Django can optimize the queries more efficiently.